### ISSUE TITLE
picovga on pimoroni board

### ISSUE THREAD

**Issue by ikjordan** [pi-gram/pico_c#1]
Hi,
I see that you are attempting to modify MCUME to use the Pimoroni Pico VGA Demo Base board with picovga. It would be wonderful if that could be achieved. However, there are comments from the author of picovga, saying that it is not possible with the correct colours. 
https://github.com/Panda381/PicoVGA/issues/1

It is also discussed in the description of the 16 colour version, where some support for the Pimoroni VGA demo board is described, but the output is green: https://breatharian.eu/hw/pico16vga/index_en.html

I see that https://github.com/fruit-bat/pico-zxspectrum supports video on a range of boards, including the VGA demo base board, and similar boards to those supported by MCUME. I wonder if there are ideas / code that could be taken from that?

**Comment by pi-gram** [comment]
Yep - I have found the same results as you mention.  The author of MCUME was suggesting making a new VGA driver or modifying hardware - which is not what I want to do, I want to just modify code.... which leads me to your 3rd URL - that looks VERY promising!  I'll take a look in a couple of days as that looks exactly like what is needed.  I'll dig through the VGA driver code they have and see if I can port it over to work properly.  THANK YOU for sharing.... I'll let you know how it goes and will update this repo with any updates.

**Comment by ikjordan** [comment]
Hi - I know it has been a while, but I started with the base MCUME ZX80/81 emulator and extended and reworked it to support the Pimoroni VGA and DVI boards. The finished code is at:
https://github.com/ikjordan/picozx81
